src/supertux/game_session.hpp

```diff
@@ -76,6 +76,13 @@ class GameSession final : public Screen,
   };
 
 public:
+  enum class FadeType
+  {
+    NONE,
+    FADE,
+    CIRCLE
+  };
```

Comment:
Maybe these enumerators would be better suited in the `ScreenFade` class ("src/supertux/screen_fade.hpp)?

Reply:
Yes, I will move it there.